src/index.scss

```diff
@@ -285,6 +285,11 @@ $collapseHoverColor: #353535;
     &:last-child {
         text-align: right !important;
     }
+
+    &--duration {
```

Comment:
I can't say I'm against it but it does not feel right for me, is there any reason we use the data attributes for styling here?